sqlmesh/core/context.py

```diff
@@ -848,12 +848,13 @@ def create_external_models(self) -> None:
         The schema file contains all columns and types of external models, allowing for more robust
         lineage, validation, and optimizations.
         """
+        models = self._models or self._loader.load(self, update_schemas=False).models
```

Comment:
So it seems like in this case we only need to load models but we'd load everything instead.  

If we made `_load_models` public and kept everything else unchanged we'd only load models and we wouldn't require the `update_schemas` flag.

Reply:
I looked at that first too. _load_models requires macros, etc., So it felt unnecessarily complicated to load each bit manually to save time on a one-off command.